src/comp/createProjPage/ProjectBasicInfoForm.tsx

```diff
@@ -37,40 +37,47 @@ class ProjectBasicInfoForm extends React.Component<
 
   handleNameInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     // multiline属性を消すと表示が乱れるのでこっちで複数行入力を阻止する
-    if (e.target.value.endsWith("\n")) return;
+    if (e.target.value.indexOf("\n") !== -1) return;
```

Comment:
```suggestion
    if (e.target.value.includes("\n")) return;
```
`includes` の方がいいと思います
他のところも同様に直してほしいです